charts/kyverno/templates/cleanup-controller/_helpers.tpl

```diff
@@ -0,0 +1,41 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Shouldn't this file be included in the upstream chart? Put another way, why are we packaging this or other files from the upstream chart, here?

Reply:
Thanks, I'm OK with the structure as long as you've templated the chart out with the upstream in place and it passes your validation.